docs/lcdproc-dev/language.docbook

```diff
@@ -401,6 +401,49 @@
 	  </listitem>
 	</varlistentry>
 
+	<varlistentry>
+	  <term>
+	    <command>key_add
+	      <option><replaceable>screen_id</replaceable></option>
+	      <option><replaceable>key</replaceable></option>...
+	    </command>
+	    since protocol version 0.4
+	  </term>
+	  <listitem>
+	    <para>
+	      Reserves key(s) for interaction of the user with the screen.
+	      Reservations with this command always take precedence over
+	      reservations with <command>client_add_key</command>, but are
+	      only in effect while the screen is visible on the display.
+	      The server message on key events is appended with
+	      <replaceable>screen_id</replaceable> to distinguish it from
+	      client level key events.
+	    </para>
+
+	    <para>
+	      Only one success message is generated for all keys together.
+	    </para>
+	  </listitem>
+	</varlistentry>
+
+	<varlistentry>
+	  <term>
+	    <command>key_del
+	      <option><replaceable>screen_id</replaceable></option>
+	      <option><replaceable>key</replaceable></option>...
+	    </command>
+	    since protocol version 0.4
+	  </term>
+	  <listitem>
+	    <para>
+	      Lift reservation of key(s) by <command>key_add</command>.
+	      Arguments are processed sequentially and for each one either
```

Comment:
Minor edit here to say "sequentially, and"... "generated, or..." and "message if the key".